crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -278,11 +278,19 @@ def maintenance_updates_check_errors(check)
         if check[:repositories_missing]
           ret[:repositories_missing] = {
             data: I18n.t("api.upgrade.prechecks.repos_missing.error",
-              missing: check[:repositories_missing].join(", ")),
+              missing: check[:repositories_missing]),
```

Comment:
wouldn't it be better to remove the join from the other end? other checks seem to  have join right before language code interpolation.